owncloudDomain/src/main/java/com/owncloud/android/domain/files/model/OCFileSyncInfo.kt

```diff
@@ -0,0 +1,33 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Juan Carlos Garrote Gascón
+ *
+ * Copyright (C) 2022 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.domain.files.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import java.util.UUID
+
+@Parcelize
+data class OCFileSyncInfo(
```

Comment:
If you refer to `OCFileSyncInfo.kt` and `OCFileWithSyncInfo.kt`, they are **almost** the same. In the first one we don't have info about the file, only the id, while in the second one we do have all the file information.

This was made due to the fact that we don't always need all the information about the file. For example, in the case of the previous review comment: we already have the files and we are only interested in retrieving its sync info. I think this can be useful in the future too, when we want to check if any file is synchronizing we can get a list of `OCFileSyncInfo`s instead of asking the current workers or retrieving all the file information 💡